packages/@sfx/dom-events-plugin/src/index.ts

```diff
@@ -0,0 +1 @@
+export { default as DomEventsPlugin } from './dom-events-plugin';
```

Comment:
You should be exporting the interfaces as well.

Reply:
Done.